includes/admin/settings.php

```diff
@@ -87,6 +91,45 @@ public function column_payment_method_value( $value, $affiliate ) {
 		return $value;
 	}
 
+	/**
+	 * Add a "Payment Method" column to the referrals screen.
+	 * 
+	 * @since 2.3.3
+	 *
+	 * @param array  $prepared_columns Prepared columns.
+	 * @param array  $columns  The columns for this list table.
+	 * @param object $instance List table instance.
+	 * 
+	 * @return array $prepared_columns Prepared columns.
```

Comment:
```suggestion
	 * @return array Prepared columns.
```
Remove the empty line between `@param` and `@return` notations, no variable for `@return` notations.